### PR TITLE
23.5.0

### PR DESCRIPTION
**Pull request by rich-howell**

### CODE REVIEW

**Comment by j-f1** [comment]
Can you add in the username link references at the bottom for the new contributors? `sinistersnare` `henrikmaa` `youngcw` `aaroneiche`

File: docs/Release-Notes.md [hunk 1]
```diff
@@ -2,6 +2,84 @@
...
+- [#969](https://github.com/actualbudget/actual/pull/969) Nordigen: improved error handling when bank-list API fails — thanks [MatissJanis]
+- [#972](https://github.com/actualbudget/actual/pull/972) Position notification banners always at bottom of the page — thanks [MatissJanis]
+- [#992](https://github.com/actualbudget/actual/pull/992) Allow creating transactions by clicking "enter" in the notes/payee/category field — thanks [MatissJanis]
+
+#### Maintenance
+
+- [#896](https://github.com/actualbudget/actual/pull/896) Convert few more folders in `loot-core` to Typescript — thanks [albertogasparin]
+- [#931](https://github.com/actualbudget/actual/pull/931) Cleaning up state management in autocomplete — thanks [MatissJanis]
+- [#932](https://github.com/actualbudget/actual/pull/932) Improving e2e test stability — thanks [MatissJanis]
+- [#934](https://github.com/actualbudget/actual/pull/934) Move from deprecated peg.js to Peggy for Goal template parser. — thanks [Sinistersnare]
```
Comment (j-f1) [inline docs/Release-Notes.md:50]:
```suggestion
- [#934](https://github.com/actualbudget/actual/pull/934) Move from deprecated peg.js to Peggy for Goal template parser. — thanks [sinistersnare]
```

File: docs/Release-Notes.md [hunk 2]
```diff
@@ -2,6 +2,84 @@
...
+- [#982](https://github.com/actualbudget/actual/pull/982) Run feature-request management github action only once and for the "feature" label only (not for other labels) — thanks [MatissJanis]
+- [#985](https://github.com/actualbudget/actual/pull/985) Remove unused payee rules feature — thanks [j-f1]
+
+### Actual Server 
+
+Version: 23.5.0
+
+#### Features
+
+- [#194](https://github.com/actualbudget/actual-server/pull/194) Adds support for setting and getting secrets in actual-server. Additionally the secrets can be set from client using api. — thanks [Henrikmaa]
```
Comment (j-f1) [inline docs/Release-Notes.md:69]:
```suggestion
- [#194](https://github.com/actualbudget/actual-server/pull/194) Adds support for setting and getting secrets in actual-server. Additionally the secrets can be set from client using api. — thanks [henrikmaa]
```

Comment (rich-howell) [reply docs/Release-Notes.md:69]: I didn't check these, I just copied them from the GH PR.

File: docs/Release-Notes.md (hunk 1 above)
Comment (rich-howell) [reply docs/Release-Notes.md:50]: I didn't check these, I just copied them from the GH PR.

**Comment by rich-howell** [comment]
> Can you add in the username link references at the bottom for the new contributors? `sinistersnare` `henrikmaa` `youngcw` `aaroneiche`

Done.

**Pull request merged**